src/App.js

```diff
@@ -32,7 +32,7 @@ function App() {
       const response = await axios.request(options);
       setVideoData(response.data);
       setThumbnail(response.data.picture);
-      const [{ quality, link }] = response.data.links;
+      const { quality, link } = response.data.links.find(item => item.quality.startsWith('video'));
```

Comment:
I did this because there are more links in the list, and a lot of them are audio files, and sometimes the audio is the first one in the list.